src/__tests__/e2e.js

```diff
@@ -314,13 +314,13 @@ describe("zipkin-javascript-opentracing", () => {
           ZipkinJavascriptOpentracing.FORMAT_HTTP_HEADERS,
           headers
         );
-        const newSpan = tracer.extract(
+        const newSpanCtx = tracer.extract(
```

Comment:
Would you mind adding a new e2e test with `REFERENCE_FOLLOWS_FROM`? I would like to ensure we don't break a thing. Maybe also one with both to make sure we have this clear.

Reply:
I will try doing during this week...